google-cloud-spanner/src/main/java/com/google/cloud/spanner/Options.java

```diff
@@ -51,6 +51,22 @@
   public interface ReadQueryUpdateTransactionOption
       extends ReadOption, QueryOption, UpdateOption, TransactionOption {}
 
+  /**
+   * Marker interface to mark options applicable to Create, Update and Delete operations in admin
+   * API.
+   */
+  public interface CreateUpdateDeleteAdminAPIOption
```

Comment:
Please use camel case for `Api` as well, per https://engdoc.corp.google.com/eng/doc/devguide/java/style/index.md?cl=head#s5.3-camel-case

Please change everywhere else too

Reply:
Done.